src/Http/Requests/Speaker/UpdateSpeakerRequest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Http\Requests\Speaker;
+
+use Blumilk\Meetup\Core\Http\Requests\Speaker\Rules\AvatarFileRules;
+use Blumilk\Meetup\Core\Http\Requests\Speaker\Rules\GithubLinkRules;
+use Blumilk\Meetup\Core\Http\Requests\Speaker\Rules\LinkedinLinkRules;
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpdateSpeakerRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            "name" => "required",
```

Comment:
Maybe you could use arrays of strings everywhere?

Reply:
what do you mean ?